Source/WebCore/rendering/svg/RenderSVGRect.cpp

```diff
@@ -87,22 +85,18 @@ void RenderSVGRect::updateShapeFromElement()
 
     // To decide if the stroke contains a point we create two rects which represent the inner and
     // the outer stroke borders. A stroke contains the point, if the point is between them.
```

Comment:
This comment is now out of date and I think we can remove it.

Reply:
Sounds good! Removed.